ComputeDemo/ComputeAlphaFinePointsChecks.cs

```diff
@@ -0,0 +1,43 @@
+﻿/*
```

Comment:
Side-note: it might be worth starting a new project (Google.Cloud.Compute.V1.AcceptanceTests) instead of changing this demo program. I should have suggested that before. (Shouldn't be much work though.)

Reply:
👍